README.md

```diff
@@ -266,46 +266,49 @@ class Sum extends ClientHandler<ContainerType, number, number> {
 // Server-side WebSocket setup
 async function startServer() {
     const wss = new WebSocket.Server({ port: 8080 });
-    const rpcServer = await RPCServer.createRPCServer({
-        manifest: {
-            Sum: new Sum({}),
-        },
-        logger: new Logger('rpc-server'),
-        handlerTimeoutTime: 60000,
-        idGen,
+    const rpcServer = new RPCServer({
+      logger: new Logger('rpc-server'),
+      handlerTimeoutTime: 60000,
+      idGen,
+    });
+    await rpcServer.start({
+      manifest: {
+        Sum: new Sum({}),
+      },
     });
 
     wss.on('connection', (ws) => {
-        const { readable, writable } = wsToStream(ws);
-        rpcServer.handleStream({ readable, writable, cancel: () => {} });
+      const { readable, writable } = wsToStream(ws);
+      rpcServer.handleStream({ readable, writable, cancel: () => {} });
     });
-    return { rpcServer  };
+    return { rpcServer };
 }
```

Comment:
If we're going to have examples of our agnostic RPC, does it makes sense to use a websocket in the example? @CMCDragonkai

Reply:
You shouldn't need to use websockets here. Its sufficient to do this with just regular web streams. Synthetic streams.

Reply:
Other people can figure out the rest. I'd prefer not to involve any transport protocol here at all. It confuses people. It's important the only interface required is a web stream.

So I think any ws usage should be removed.